module-1/lab-api-scavenger-game/your-code/Q2.ipynb

```diff
@@ -0,0 +1,2138 @@
+{
```

Comment:
Your code works because in this question you are counting number of records, otherwise your code would have undesired results. 
There's the same problem as in Q1, you haven't normalized your JSON and you have nested information stored in your dataframe records.